<!DOCTYPE html>
<html lang="en-us">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Your description">
  <meta name="author" content="Your name">
  <meta name="generator" content="Hugo 0.46" />
  <title>BBSRC Bistability of Cell Cycle Transitions Project</title>
  <!-- Stylesheets -->
  <link href="http://cellcycle.org.uk/css/bootstrap-v3.3.7/bootstrap.min.css" rel="stylesheet">
  
  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  
  <link type="text/css" rel="stylesheet" href="http://cellcycle.org.uk/css/materialize.css"  media="screen,projection"/>
  <link href="http://cellcycle.org.uk/css/agency.css" rel="stylesheet">


  

  <!-- Custom Fonts -->
  <link href="http://cellcycle.org.uk/font-awesome-v4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link rel="stylesheet" href="http://cellcycle.org.uk/academicons-1.7.0/css/academicons.min.css"/>


  
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700" rel="stylesheet" type="text/css">
  

  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
  <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
  <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>


  <body id="page-top" class="index">
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-fixed-top">
  <div class="container">

    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header page-scroll">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
        <a class="navbar-brand page-scroll" href="http://cellcycle.org.uk/">
          <img src="http://cellcycle.org.uk//img/bi-cycle.png" class="img-responsive" alt="bicycle">
        </a>
      

    </div>
    <!-- Collect the nav links, forms, and other content for toggling -->
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav navbar-right">
        <li class="hidden">
          <a href="http://cellcycle.org.uk/#page-top"></a>
        </li>

        

        
        <li>
          <a  href="http://cellcycle.org.uk/#about">About</a>
        </li>
        

        <li>
          <a  href="http://cellcycle.org.uk/news">News</a>
        </li>

        

        



        
        <li>
          <a href="http://cellcycle.org.uk/member">Members</a>
        </li>
        <li>
          <a href="http://cellcycle.org.uk/publication">Publications</a>
        </li>

        
          <li>
            <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html">PPSIM</a>
          </li>
        
      </ul>
    </div>
    <!-- /.navbar-collapse -->
  </div>
  <!-- /.container-fluid -->
</nav>


    <div class="portfolio-modal fade in" id="portfolioModal_stefan_heldt" tabindex="-1" role="dialog" aria-hidden="true">
      <div class="modal-content">
        <div class="container">
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <div class="modal-body">
                
                
                <h2>Stefan Heldt</h2>
                <h6 class="item-intro text-muted">Postdoc</h6>
                <img class="img-responsive img-centered img-circle" src="http://cellcycle.org.uk/img/member/stefan_heldt.jpg" alt="">
                <p>Stefan Heldt studied Biosystems Engineering at the Otto von Guericke University in Magdeburg, Germany. He received his degree in 2009 after completing a diploma thesis on stochastic simulations of mRNA translation in the group of Prof. Celso Grebogi at the University of Aberdeen, Scotland. He then joined the Max Planck Institute for Dynamics of Complex Technical Systems for a Ph.D. developing mechanistic, multi-scale models of influenza virus infection. This work awarded him the 2016 MTZ Award for Medical Systems Biology. Since 2015, he is a postdoctoral research fellow in Bela Novak’s lab, where he uses mathematical modelling to understand the dynamic characteristics of the mammalian cell cycle and its regulation. In this project, he is working closely with Alexis Barr from Chris Bakal’s lab.</p>
                <ul class="list-inline social-buttons">
                  
                    
                  
                    
                    
                      <li><a href="https://scholar.google.co.uk/citations?user=eWX_y28AAAAJ&amp;hl=en&amp;oi=ao" target="_blank"><i class="ai ai-google-scholar"></i></a></li>
                    
                  
                    
                    
                      <li><a href="https://www.linkedin.com/in/stefan-heldt-99067210/" target="_blank"><i class="fa fa-linkedin"></i></a></li>
                    
                  
                    
                    
                      <li><a href="https://www.researchgate.net/profile/Frank_Heldt" target="_blank"><i class="ai ai-researchgate"></i></a></li>
                    
                  
                    
                    
                      <li><a href="https://twitter.com/FS_Heldt" target="_blank"><i class="fa fa-twitter"></i></a></li>
                    
                  
                    
                  
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div class="container" id="publications">
          <div class="row">
            <div class="col-lg-12 text-center">
              <h4 class="section-heading">Publications contributed to in the context of this project</h4>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <ul class="timeline">

                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_CurrBiol_2019.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Jan 9, 2020</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/heldt_currbiol_2019/"><h4>A single light-responsive sizer can control multiple-fission cycles in Chlamydomonas</h4></a>
      <h5 class="subheading">
        Stefan Heldt, John J. Tyson, Frederick R. Cross, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2019.12.026" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2019.12.026</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2019_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2019_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Most eukaryotic cells execute binary division after each mass doubling in order to maintain size homeostasis by coordinating cell growth and division. By contrast, the photosynthetic green alga Chlamydomonas can grow more than 8-fold during daytime and then, at night, undergo rapid cycles of DNA replication, mitosis, and cell division, producing up to 16 daughter cells. Here, we propose a mechanistic model for multiple-fission cycles and cell-size control in Chlamydomonas. The model comprises a light-sensitive and size-dependent biochemical toggle switch that acts as a sizer, guarding transitions into and exit from a phase of cell-division cycle oscillations. This simple “sizer-oscillator” arrangement reproduces the experimentally observed features of multiple-fission cycles and the response of Chlamydomonas cells to different light-dark regimes. Our model also makes specific predictions about the size dependence of the time of onset of cell division after cells are transferred from light to dark conditions, and we confirm these predictions by single-cell experiments. Collectively, our results provide a new perspective on the concept of a “commitment point” during the growth of Chlamydomonas cells and hint at intriguing similarities of cell-size control in different eukaryotic lineages.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_PlosComputBiol_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Oct 24, 2018</h6>
      <h5 class="subheading">PLOS Computational Biology</h5>
      <a href="/publication/heldt_plos_2018/"><h4>Dilution and titration of cell-cycle regulators may control cell size in budding yeast</h4></a>
      <h5 class="subheading">
        Stefan Heldt, Reece Lunstone, John J. Tyson, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1371/journal.pcbi.1006548" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1371/journal.pcbi.1006548</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2018b_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2018b_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The size of a cell sets the scale for all biochemical processes within it, thereby affecting cellular fitness and survival. Hence, cell size needs to be kept within certain limits and relatively constant over multiple generations. However, how cells measure their size and use this information to regulate growth and division remains controversial. Here, we present two mechanistic mathematical models of the budding yeast (S. cerevisiae) cell cycle to investigate competing hypotheses on size control: inhibitor dilution and titration of nuclear sites. Our results suggest that an inhibitor-dilution mechanism, in which cell growth dilutes the transcriptional inhibitor Whi5 against the constant activator Cln3, can facilitate size homeostasis. This is achieved by utilising a positive feedback loop to establish a fixed size threshold for the START transition, which efficiently couples cell growth to cell cycle progression. Yet, we show that inhibitor dilution cannot reproduce the size of mutants that alter the cell’s overall ploidy and WHI5 gene copy number. By contrast, size control through titration of Cln3 against a constant number of genomic binding sites for the transcription factor SBF recapitulates both size homeostasis and the size of these mutant strains. Moreover, this model produces an imperfect ‘sizer’ behaviour in G1 and a ‘timer’ in S/G2/M, which combine to yield an ‘adder’ over the whole cell cycle; an observation recently made in experiments. Hence, our model connects these phenomenological data with the molecular details of the cell cycle, providing a systems-level perspective of budding yeast size control.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Novak_CurrOpin_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Feb 22, 2018</h6>
      <h5 class="subheading">Current Opinion in Systems Biology</h5>
      <a href="/publication/novak_curropinsysbiol_2018/"><h4>Genome Stability during Cell Proliferation: A Systems Analysis of the Molecular Mechanisms Controlling Progression through the Eukaryotic Cell Cycle</h4></a>
      <h5 class="subheading">
        Bela Novak, Stefan Heldt, John J. Tyson
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.coisb.2018.02.004" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.coisb.2018.02.004</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Well-nourished cells in a favorable environment (well supplied with growth factors, cytokines, and/or hormones and free from stresses, ionizing radiation, etc.) will grow, replicate their genome, and divide into two daughter cells, fully prepared to repeat the process. This cycle of DNA replication and division underlies all aspects of biological growth, reproduction, repair and development. As such, it is essential that the cell’s genome be guarded against damage during the replication/division process, lest the error(s) be irrevocably passed down to all future generations of progeny. Hence, cell cycle progression is closely guarded against major sources of errors, in particular DNA damage and misalignment of replicated chromosomes on the mitotic spindle. In this review article we examine closely the molecular mechanisms that maintain genomic integrity during the cell division cycle, and we find an unexpected and intriguing arrangement of concatenated and nested bistable toggle switches. The topology of the network seems to play crucial roles in maintaining the stability of the genome during cell proliferation.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_Pnas_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Feb 21, 2018</h6>
      <h5 class="subheading">Proceedings of the National Academy of Sciences</h5>
      <a href="/publication/heldt_pnas_2018/"><h4>A comprehensive model for the proliferation-quiescence decision in response to endogenous DNA damage in human cells</h4></a>
      <h5 class="subheading">
        Stefan Heldt, Alexis Barr, Sam Cooper, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1073/pnas.1715345115" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1073/pnas.1715345115</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2018_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2018_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Human cells that suffer mild DNA damage can enter a reversible state of growth arrest known as quiescence. This
decision to temporarily exit the cell cycle is essential to prevent the propagation of mutations and most cancer cells harbour
defects in the underlying control system. Here we present a mechanistic mathematical model to study the proliferation-quiescence
decision in non-transformed human cells. We show that two bistable switches, the restriction point (RP) and the G1/S transition,
mediate this decision by integrating DNA damage and mitogen signals. In particular, our data suggests that the cyclin-dependent
kinase inhibitor p21 (Cip1/Waf1), which is expressed in response to DNA damage, promotes quiescence by blocking positive feedback
loops that facilitate G1 progression downstream of serum stimulation. Intriguingly, cells exploit bistability in the RP to convert
graded p21 and mitogen signals into an all-or-nothing cell cycle response. The same mechanism creates a window of opportunity
where G1 cells that have passed the RP can revert to quiescence if exposed to DNA damage. We present experimental evidence that
cells gradually lose this ability to revert to quiescence as they progress through G1 and that the onset of rapid p21 degradation
at the G1/S transition prevents this response altogether, insulating S-phase from mild, endogenous DNA damage. Thus, two bistable
switches conspire in the early cell cycle to provide both sensitivity and robustness to external stimuli.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_naturecommunications_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Mar 20, 2017</h6>
      <h5 class="subheading">Nature Communications</h5>
      <a href="/publication/barr_naturecommunications_2017/"><h4>DNA damage during S-phase mediates the proliferation-quiescence decision in the subsequent G1 via p21 expression</h4></a>
      <h5 class="subheading">
        Alexis Barr, Samuel Cooper, Stefan Heldt, Francesca Butera, Henriette Stoy, Jörg Mansfeld, Bela Novak, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1038/ncomms14728" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1038/ncomms14728</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2017_Barr_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2017_Barr_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Following DNA damage caused by exogenous sources, such as ionising radiation, the tumour suppressor p53 mediates cell cycle arrest via expression of the CDK inhibitor, p21. However, the role of p21 in maintaining genomic stability in the absence of exogenous DNA damaging agents is unclear. Here, using live single cell measurements of p21 protein in proliferating cultures, we show that naturally-occurring DNA damage incurred over S-phase causes p53-dependent accumulation of p21 during mother G2- and daughter G1-phases. High p21 levels mediate G1 arrest via CDK inhibition, yet lower levels have no impact on G1 progression, and the ubiquitin ligases CRL4<sup>Cdt2</sup> and SCF<sup>Skp2</sup> couple to degrade p21 prior to the G1/S transition. Mathematical modelling reveals that a bistable switch, created by CRL4<sup>Cdt2</sup>, promotes irreversible S-phase entry by keeping p21 levels low, preventing premature S-phase exit upon DNA damage. Thus, we characterise how p21 regulates the proliferation-quiescence decision to maintain genomic stability.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_cellsystems_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 27, 2016</h6>
      <h5 class="subheading">Cell Systems</h5>
      <a href="/publication/barr_cellsystems_2016/"><h4>A Dynamical Framework for the All-or-None G1/S Transition</h4></a>
      <h5 class="subheading">
        Alexis Barr, Stefan Heldt, Tongli Zhang, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cels.2016.01.001" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cels.2016.01.001</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The transition from G1 into DNA replication (S phase) is an emergent behavior resulting from dynamic and complex interactions between cyclin-dependent kinases (Cdks), Cdk inhibitors (CKIs), and the anaphase-promoting complex/cyclosome (APC/C). Understanding the cellular decision to commit to S phase requires a quantitative description of these interactions. We apply quantitative imaging of single human cells to track the expression of G1/S regulators and use these data to parametrize a stochastic mathematical model of the G1/S transition. We show that a rapid, proteolytic, double-negative feedback loop between Cdk2:Cyclin and the Cdk inhibitor p27Kip1 drives a switch-like entry into S phase. Furthermore, our model predicts that increasing Emi1 levels throughout S phase are critical in maintaining irreversibility of the G1/S transition, which we validate using Emi1 knockdown and live imaging of G1/S reporters. This work provides insight into the general design principles of the signaling networks governing the temporally abrupt transitions between cell-cycle phases.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>

    <!-- jQuery -->
<script src="http://cellcycle.org.uk/js/jquery-v3.3.1/jquery.min.js"></script>

<!-- Bootstrap Core JavaScript -->
<script src="http://cellcycle.org.uk/js/bootstrap-v3.3.7/bootstrap.js"></script>

<!-- Plugin JavaScript -->
<script src="http://cellcycle.org.uk/js/classie-v1.0.1.js"></script>
<script src="http://cellcycle.org.uk/js/cbpAnimatedHeader-v1.0.0.js"></script>

<!-- Contact Form JavaScript -->
<script src="http://cellcycle.org.uk/js/jqBootstrapValidation-v1.3.6-dirty.js"></script>
<script src="http://cellcycle.org.uk/js/contact_me.js"></script>

<!-- Custom Theme JavaScript -->
<script src="http://cellcycle.org.uk/js/agency.js"></script>

<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
<script type="text/javascript" src="http://cellcycle.org.uk/js/materialize.min.js"></script>







  </body>
</html>
