<!DOCTYPE html>
<html lang="en-us">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Your description">
  <meta name="author" content="Your name">
  <meta name="generator" content="Hugo 0.46" />
  <title>BBSRC Bistability of Cell Cycle Transitions Project</title>
  <!-- Stylesheets -->
  <link href="http://cellcycle.org.uk/css/bootstrap-v3.3.7/bootstrap.min.css" rel="stylesheet">
  
  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  
  <link type="text/css" rel="stylesheet" href="http://cellcycle.org.uk/css/materialize.css"  media="screen,projection"/>
  <link href="http://cellcycle.org.uk/css/agency.css" rel="stylesheet">


  

  <!-- Custom Fonts -->
  <link href="http://cellcycle.org.uk/font-awesome-v4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link rel="stylesheet" href="http://cellcycle.org.uk/academicons-1.7.0/css/academicons.min.css"/>


  
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700" rel="stylesheet" type="text/css">
  

  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
  <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
  <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>


  <body id="page-top" class="index">
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-fixed-top">
  <div class="container">

    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header page-scroll">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
        <a class="navbar-brand page-scroll" href="http://cellcycle.org.uk/">
          <img src="http://cellcycle.org.uk//img/bi-cycle.png" class="img-responsive" alt="bicycle">
        </a>
      

    </div>
    <!-- Collect the nav links, forms, and other content for toggling -->
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav navbar-right">
        <li class="hidden">
          <a href="http://cellcycle.org.uk/#page-top"></a>
        </li>

        

        
        <li>
          <a  href="http://cellcycle.org.uk/#about">About</a>
        </li>
        

        <li>
          <a  href="http://cellcycle.org.uk/news">News</a>
        </li>

        

        



        
        <li>
          <a href="http://cellcycle.org.uk/member">Members</a>
        </li>
        <li>
          <a href="http://cellcycle.org.uk/publication">Publications</a>
        </li>

        
          <li>
            <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html">PPSIM</a>
          </li>
        
      </ul>
    </div>
    <!-- /.navbar-collapse -->
  </div>
  <!-- /.container-fluid -->
</nav>


    <div class="portfolio-modal fade in" id="portfolioModal_alexis_barr" tabindex="-1" role="dialog" aria-hidden="true">
      <div class="modal-content">
        <div class="container">
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <div class="modal-body">
                
                
                <h2>Alexis Barr</h2>
                <h6 class="item-intro text-muted">Postdoc</h6>
                <img class="img-responsive img-centered img-circle" src="http://cellcycle.org.uk/img/member/alexis_barr.jpg" alt="">
                <p>Alexis Barr studied Biological Natural Sciences at the University of Cambridge, specialising in Biochemistry in her third and fourth years.  She carried out her PhD in centrosome biology at the CRUK Cambridge Research Institute under the supervision of Fanni Gergely.  During that time she used genetic engineering, molecular biology and live cell imaging to study centrosome function and microtubule dynamics during mitotic spindle assembly in vertebrate cells. In 2010, Alexis moved to the Institute of Cancer Research (ICR), London, to join Chris Bakal&rsquo;s team for her Postdoctoral research, in order to take a systems-level approach to analyse the control of the G1/S transition.  She is developing tools and cell lines to analyse the signalling events around the G1/S transition in real-time, at the single cell level.  She is working closely with Stefan Heldt in Bela Novak&rsquo;s team towards building a comprehensive model of the G1/S transition in human cells.</p>
                <ul class="list-inline social-buttons">
                  
                    
                  
                    
                    
                      <li><a href="https://scholar.google.co.uk/citations?hl=en&amp;view_op=list_works&amp;gmla=AJsN-F6r5qxEYyoGdiO7E2v8d3g073TH_pWaubhs4FloVQARgl2MdRwExLOl__ELBdPpHaUqa9Sti-_rjzxMhnMM9K45skSBWQ&amp;user=ixJJ57UAAAAJ" target="_blank"><i class="ai ai-google-scholar"></i></a></li>
                    
                  
                    
                  
                    
                    
                      <li><a href="https://www.researchgate.net/profile/Alexis_Barr2" target="_blank"><i class="ai ai-researchgate"></i></a></li>
                    
                  
                    
                  
                    
                  
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div class="container" id="publications">
          <div class="row">
            <div class="col-lg-12 text-center">
              <h4 class="section-heading">Publications contributed to in the context of this project</h4>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <ul class="timeline">

                
                
                
                
                
                
                
                
                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_Pnas_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Feb 21, 2018</h6>
      <h5 class="subheading">Proceedings of the National Academy of Sciences</h5>
      <a href="/publication/heldt_pnas_2018/"><h4>A comprehensive model for the proliferation-quiescence decision in response to endogenous DNA damage in human cells</h4></a>
      <h5 class="subheading">
        Stefan Heldt, Alexis Barr, Sam Cooper, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1073/pnas.1715345115" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1073/pnas.1715345115</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2018_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2018_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Human cells that suffer mild DNA damage can enter a reversible state of growth arrest known as quiescence. This
decision to temporarily exit the cell cycle is essential to prevent the propagation of mutations and most cancer cells harbour
defects in the underlying control system. Here we present a mechanistic mathematical model to study the proliferation-quiescence
decision in non-transformed human cells. We show that two bistable switches, the restriction point (RP) and the G1/S transition,
mediate this decision by integrating DNA damage and mitogen signals. In particular, our data suggests that the cyclin-dependent
kinase inhibitor p21 (Cip1/Waf1), which is expressed in response to DNA damage, promotes quiescence by blocking positive feedback
loops that facilitate G1 progression downstream of serum stimulation. Intriguingly, cells exploit bistability in the RP to convert
graded p21 and mitogen signals into an all-or-nothing cell cycle response. The same mechanism creates a window of opportunity
where G1 cells that have passed the RP can revert to quiescence if exposed to DNA damage. We present experimental evidence that
cells gradually lose this ability to revert to quiescence as they progress through G1 and that the onset of rapid p21 degradation
at the G1/S transition prevents this response altogether, insulating S-phase from mild, endogenous DNA damage. Thus, two bistable
switches conspire in the early cell cycle to provide both sensitivity and robustness to external stimuli.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/cooper_bioinformatics_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Tuesday, Jun 20, 2017</h6>
      <h5 class="subheading">Bioinformatics</h5>
      <a href="/publication/cooper_bioinformatics_2017/"><h4>NucliTrack: An integrated nuclei tracking application</h4></a>
      <h5 class="subheading">
        Sam Cooper, Alexis Barr, Robert Glen, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1093/bioinformatics/btx404" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1093/bioinformatics/btx404</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/samocooper/nuclitrack" target="_blank" title="View Code Resource on Github">https://github.com/samocooper/nuclitrack</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Live imaging studies give unparalleled insight into dynamic single cell behaviours and fate decisions. However, the challenge of reliably tracking single cells over long periods of time limits both the throughput and ease with which such studies can be performed. Here, we present NucliTrack, a cross platform solution for automatically segmenting, tracking and extracting features from fluorescently-labelled nuclei. NucliTrack performs similarly to other state-of-the-art cell tracking algorithms, but NucliTrack’s interactive, graphical interface makes it significantly more user friendly.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_naturecommunications_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Mar 20, 2017</h6>
      <h5 class="subheading">Nature Communications</h5>
      <a href="/publication/barr_naturecommunications_2017/"><h4>DNA damage during S-phase mediates the proliferation-quiescence decision in the subsequent G1 via p21 expression</h4></a>
      <h5 class="subheading">
        Alexis Barr, Samuel Cooper, Stefan Heldt, Francesca Butera, Henriette Stoy, Jörg Mansfeld, Bela Novak, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1038/ncomms14728" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1038/ncomms14728</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2017_Barr_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2017_Barr_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Following DNA damage caused by exogenous sources, such as ionising radiation, the tumour suppressor p53 mediates cell cycle arrest via expression of the CDK inhibitor, p21. However, the role of p21 in maintaining genomic stability in the absence of exogenous DNA damaging agents is unclear. Here, using live single cell measurements of p21 protein in proliferating cultures, we show that naturally-occurring DNA damage incurred over S-phase causes p53-dependent accumulation of p21 during mother G2- and daughter G1-phases. High p21 levels mediate G1 arrest via CDK inhibition, yet lower levels have no impact on G1 progression, and the ubiquitin ligases CRL4<sup>Cdt2</sup> and SCF<sup>Skp2</sup> couple to degrade p21 prior to the G1/S transition. Mathematical modelling reveals that a bistable switch, created by CRL4<sup>Cdt2</sup>, promotes irreversible S-phase entry by keeping p21 levels low, preventing premature S-phase exit upon DNA damage. Thus, we characterise how p21 regulates the proliferation-quiescence decision to maintain genomic stability.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_cellsystems_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 27, 2016</h6>
      <h5 class="subheading">Cell Systems</h5>
      <a href="/publication/barr_cellsystems_2016/"><h4>A Dynamical Framework for the All-or-None G1/S Transition</h4></a>
      <h5 class="subheading">
        Alexis Barr, Stefan Heldt, Tongli Zhang, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cels.2016.01.001" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cels.2016.01.001</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The transition from G1 into DNA replication (S phase) is an emergent behavior resulting from dynamic and complex interactions between cyclin-dependent kinases (Cdks), Cdk inhibitors (CKIs), and the anaphase-promoting complex/cyclosome (APC/C). Understanding the cellular decision to commit to S phase requires a quantitative description of these interactions. We apply quantitative imaging of single human cells to track the expression of G1/S regulators and use these data to parametrize a stochastic mathematical model of the G1/S transition. We show that a rapid, proteolytic, double-negative feedback loop between Cdk2:Cyclin and the Cdk inhibitor p27Kip1 drives a switch-like entry into S phase. Furthermore, our model predicts that increasing Emi1 levels throughout S phase are critical in maintaining irreversibility of the G1/S transition, which we validate using Emi1 knockdown and live imaging of G1/S reporters. This work provides insight into the general design principles of the signaling networks governing the temporally abrupt transitions between cell-cycle phases.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>

    <!-- jQuery -->
<script src="http://cellcycle.org.uk/js/jquery-v3.3.1/jquery.min.js"></script>

<!-- Bootstrap Core JavaScript -->
<script src="http://cellcycle.org.uk/js/bootstrap-v3.3.7/bootstrap.js"></script>

<!-- Plugin JavaScript -->
<script src="http://cellcycle.org.uk/js/classie-v1.0.1.js"></script>
<script src="http://cellcycle.org.uk/js/cbpAnimatedHeader-v1.0.0.js"></script>

<!-- Contact Form JavaScript -->
<script src="http://cellcycle.org.uk/js/jqBootstrapValidation-v1.3.6-dirty.js"></script>
<script src="http://cellcycle.org.uk/js/contact_me.js"></script>

<!-- Custom Theme JavaScript -->
<script src="http://cellcycle.org.uk/js/agency.js"></script>

<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
<script type="text/javascript" src="http://cellcycle.org.uk/js/materialize.min.js"></script>







  </body>
</html>
