<!DOCTYPE html>
<html lang="en-us">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Your description">
  <meta name="author" content="Your name">
  <meta name="generator" content="Hugo 0.46" />
  <title>BBSRC Bistability of Cell Cycle Transitions Project</title>
  <!-- Stylesheets -->
  <link href="http://cellcycle.org.uk/css/bootstrap-v3.3.7/bootstrap.min.css" rel="stylesheet">
  
  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  
  <link type="text/css" rel="stylesheet" href="http://cellcycle.org.uk/css/materialize.css"  media="screen,projection"/>
  <link href="http://cellcycle.org.uk/css/agency.css" rel="stylesheet">


  

  <!-- Custom Fonts -->
  <link href="http://cellcycle.org.uk/font-awesome-v4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link rel="stylesheet" href="http://cellcycle.org.uk/academicons-1.7.0/css/academicons.min.css"/>


  
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700" rel="stylesheet" type="text/css">
  

  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
  <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
  <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>


  <body id="page-top" class="index">
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-fixed-top">
  <div class="container">

    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header page-scroll">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
        <a class="navbar-brand page-scroll" href="http://cellcycle.org.uk/">
          <img src="http://cellcycle.org.uk//img/bi-cycle.png" class="img-responsive" alt="bicycle">
        </a>
      

    </div>
    <!-- Collect the nav links, forms, and other content for toggling -->
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav navbar-right">
        <li class="hidden">
          <a href="http://cellcycle.org.uk/#page-top"></a>
        </li>

        

        
        <li>
          <a  href="http://cellcycle.org.uk/#about">About</a>
        </li>
        

        <li>
          <a  href="http://cellcycle.org.uk/news">News</a>
        </li>

        

        



        
        <li>
          <a href="http://cellcycle.org.uk/member">Members</a>
        </li>
        <li>
          <a href="http://cellcycle.org.uk/publication">Publications</a>
        </li>

        
          <li>
            <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html">PPSIM</a>
          </li>
        
      </ul>
    </div>
    <!-- /.navbar-collapse -->
  </div>
  <!-- /.container-fluid -->
</nav>


    <section id="publications">
      <div class="container">
        <div class="row">
          <div class="col-lg-12 text-center">
            <h2 class="section-heading">Publications</h2>
            <h3 class="section-subheading text-muted">Peer-reviewed publications published in the context of this project</h3>
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-lg-8 col-lg-offset-2">
          <ul class="timeline">
            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/novaktyson_mbc_2021.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Feb 3, 2021</h6>
      <h5 class="subheading">Molecular Biology of the Cell</h5>
      <a href="/publication/novaktyson_mbc_2021/"><h4>Computational Modelling of Chromosome Re-Replication in Mutant Strains of Fission Yeast</h4></a>
      <h5 class="subheading">
        Bela Novak, John J. Tyson
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1091/mbc.E20-09-0610" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1091/mbc.E20-09-0610</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Typically cells replicate their genome only once per division cycle, but under some
circumstances, both natural and unnatural, cells synthesize an overabundance of DNA, either in
a disorganized fashion (&lsquo;over-replication&rsquo;) or by a systematic doubling of chromosome number
(&lsquo;endoreplication&rsquo;). These variations on the theme of DNA replication and division have been
studied in strains of fission yeast, <em>Schizosaccharomyces pombe</em>, carrying mutations that
interfere with the function of mitotic cyclin-dependent kinase (Cdk1:Cdc13) without impeding the
roles of DNA-replication loading-factor (Cdc18) and S-phase cyclin-dependent kinase
(Cdk1:Cig2). Some of these mutations support endoreplication, and some over-replication. In
this paper, we propose a dynamical model of the interactions among the proteins governing
DNA replication and cell division in fission yeast. By computational simulations of the
mathematical model, we account for the observed phenotypes of these re-replicating mutants,
and by theoretical analysis of the dynamical system, we provide insight into the molecular
distinctions between over-replicating and endoreplicating cells. In case of induced over-
production of regulatory proteins, our model predicts that cells first switch from normal mitotic
cell cycles to growth-controlled endoreplication, and ultimately to disorganized over-replication,
parallel to the slow increase of protein to very high levels.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/novaktyson_curropcellbiol_2021.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Jan 4, 2021</h6>
      <h5 class="subheading">Current Opinion in Cell Biology</h5>
      <a href="/publication/novaktyson_curropcellbiol_2021-copy/"><h4>Mechanisms of Signalling-Memory Governing Progression Through the Eukaryotic Cell Cycle</h4></a>
      <h5 class="subheading">
        Bela Novak, John J. Tyson
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.ceb.2020.12.003" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.ceb.2020.12.003</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>As cells pass through each replication-division cycle, they must be able to postpone further
progression if they detect any threats to genome integrity, such as DNA damage or misaligned
chromosomes. Once a &#39;decision&#39; is made to proceed, the cell unequivocally enters into a
qualitatively different biochemical state, which makes the transitions from one cell cycle phase
to the next switch-like and irreversible. Each transition is governed by a unique signalling
network; nonetheless, they share a common characteristic of bistable behaviour, a hallmark of
molecular memory devices. Comparing the cell cycle signalling mechanisms acting at the
restriction point, G1/S, G2/M and meta-to-anaphase transitions, we deduce a generic network
motif of coupled positive and negative feedback loops underlying each transition.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/novaktyson_trendscellbiol_2020.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Apr 30, 2020</h6>
      <h5 class="subheading">Trends in Cell Biology</h5>
      <a href="/publication/novaktyson_trendscellbiol_2020/"><h4>A Dynamical Paradigm for Molecular Cell Biology</h4></a>
      <h5 class="subheading">
        Bela Novak, John J. Tyson
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.tcb.2020.04.002" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.tcb.2020.04.002</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The driving passion of molecular cell biologists is to understand the molecular mechanisms that
control important aspects of cell physiology, but this ambition is often limited by the wealth of
molecular details currently known about these mechanisms. Their complexity overwhelms our
intuitive notions of how molecular regulatory networks might respond under normal and stressful
conditions. To make progress we need a new paradigm for connecting molecular biology to cell
physiology. We suggest an approach that uses precise mathematical methods to associate the
qualitative features of dynamical systems, as conveyed by &lsquo;bifurcation diagrams&rsquo;, with &lsquo;signal-
response&rsquo; curves measured by cell biologists.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/hegarat_emboj_2020.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Apr 30, 2020</h6>
      <h5 class="subheading">EMBO Journal</h5>
      <a href="/publication/hegarat_emboj_2020/"><h4>Cyclin A Triggers Mitosis Either via the Greatwall Kinase Pathway or Cyclin B</h4></a>
      <h5 class="subheading">
        Nadia Hegarat, Adrijana Crncec, Maria F Suarez Peredo Rodriguez, Fabio Echegaray Iturra, Yan Gu, Oliver Busby, Paul F Lang, Alexis R Barr, Chris Bakal, Masato T Kanemaki, Angus I Lamond, Bela Novak, Tony Ly, Helfrid Hochegger
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.15252/embj.2020104419" target="_blank" title="Visit publisher website">https://dx.doi.org/10.15252/embj.2020104419</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Two mitotic cyclin types, cyclin A and B, exist in higher eukaryotes, but their specialised
functions in mitosis are incompletely understood. Using degron tags for rapid inducible protein
removal, we analyse how acute depletion of these proteins affects mitosis. Loss of cyclin A in
G2-phase prevents mitotic entry. Cells lacking cyclin B can enter mitosis and phosphorylate
most mitotic proteins, because of parallel PP2A:B55 phosphatase inactivation by Greatwall
kinase. The final barrier to mitotic establishment corresponds to nuclear envelope breakdown,
which requires a decisive shift in the balance of cyclin-dependent kinase Cdk1 and PP2A:B55
activity. Beyond this point, cyclin B/Cdk1 is essential for phosphorylation of a distinct subset of
mitotic Cdk1 substrates that are essential to complete cell division. Our results identify how
cyclin A, cyclin B and Greatwall kinase coordinate mitotic progression by increasing levels of
Cdk1-dependent substrate phosphorylation.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_CurrBiol_2019.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Jan 9, 2020</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/heldt_currbiol_2019/"><h4>A single light-responsive sizer can control multiple-fission cycles in Chlamydomonas</h4></a>
      <h5 class="subheading">
        Stefan Heldt, John J. Tyson, Frederick R. Cross, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2019.12.026" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2019.12.026</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2019_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2019_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Most eukaryotic cells execute binary division after each mass doubling in order to maintain size homeostasis by coordinating cell growth and division. By contrast, the photosynthetic green alga Chlamydomonas can grow more than 8-fold during daytime and then, at night, undergo rapid cycles of DNA replication, mitosis, and cell division, producing up to 16 daughter cells. Here, we propose a mechanistic model for multiple-fission cycles and cell-size control in Chlamydomonas. The model comprises a light-sensitive and size-dependent biochemical toggle switch that acts as a sizer, guarding transitions into and exit from a phase of cell-division cycle oscillations. This simple “sizer-oscillator” arrangement reproduces the experimentally observed features of multiple-fission cycles and the response of Chlamydomonas cells to different light-dark regimes. Our model also makes specific predictions about the size dependence of the time of onset of cell division after cells are transferred from light to dark conditions, and we confirm these predictions by single-cell experiments. Collectively, our results provide a new perspective on the concept of a “commitment point” during the growth of Chlamydomonas cells and hint at intriguing similarities of cell-size control in different eukaryotic lineages.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Hayward_Jcb_2019.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 23, 2019</h6>
      <h5 class="subheading">Journal of Cell Biology</h5>
      <a href="/publication/hayward_jcb_2019/"><h4>CDK1-CCNB1 creates a spindle checkpoint–permissive state by enabling MPS1 kinetochore localization</h4></a>
      <h5 class="subheading">
        Daniel Hayward, Tatiana Alfonso-Pérez, Michael J. Cundell, Michael Hopkins, James Holder, James Bancroft, Lukas H. Hutter, Bela Novak, Francis A. Barr, Ulrike Gruneberg
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1083/jcb.201808014" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1083/jcb.201808014</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Spindle checkpoint signaling is initiated by recruitment of the kinase MPS1 to unattached kinetochores during mitosis. We show that CDK1-CCNB1 and a counteracting phosphatase PP2A-B55 regulate the engagement of human MPS1 with unattached kinetochores by controlling the phosphorylation status of S281 in the kinetochore-binding domain. This regulation is essential for checkpoint signaling, since MPS1S281A is not recruited to unattached kinetochores and fails to support the recruitment of other checkpoint proteins. Directly tethering MPS1S281A to the kinetochore protein Mis12 bypasses this regulation and hence the requirement for S281 phosphorylation in checkpoint signaling. At the metaphase–anaphase transition, MPS1 S281 dephosphorylation is delayed because PP2A-B55 is negatively regulated by CDK1-CCNB1 and only becomes fully active once CCNB1 concentration falls below a characteristic threshold. This mechanism prolongs the checkpoint-responsive period when MPS1 can localize to kinetochores and enables a response to late-stage spindle defects. By acting together, CDK1-CCNB1 and PP2A-B55 thus create a spindle checkpoint–permissive state and ensure the fidelity of mitosis.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/AlfonsoPerez_Jcb_2019.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 23, 2019</h6>
      <h5 class="subheading">Journal of Cell Biology</h5>
      <a href="/publication/alfonsoperez_jcb_2019/"><h4>MAD1-dependent recruitment of CDK1-CCNB1 to kinetochores promotes spindle checkpoint signaling</h4></a>
      <h5 class="subheading">
        Tatiana Alfonso-Pérez, Daniel Hayward, James Holder, Ulrike Gruneberg, Francis A. Barr
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1083/jcb.201808015" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1083/jcb.201808015</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Cyclin B–dependent kinase (CDK1-CCNB1) promotes entry into mitosis. Additionally, it inhibits mitotic exit by activating the spindle checkpoint. This latter role is mediated through phosphorylation of the checkpoint kinase MPS1 and other spindle checkpoint proteins. We find that CDK1-CCNB1 localizes to unattached kinetochores and like MPS1 is lost from these structures upon microtubule attachment. This suggests that CDK1-CCNB1 is an integral component and not only an upstream regulator of the spindle checkpoint pathway. Complementary proteomic and cell biological analysis demonstrate that the spindle checkpoint protein MAD1 is one of the major components of CCNB1 complexes, and that CCNB1 is recruited to unattached kinetochores in an MPS1-dependent fashion through interaction with the first 100 amino acids of MAD1. This MPS1 and MAD1-dependent pool of CDK1-CCNB1 creates a positive feedback loop necessary for timely recruitment of MPS1 to kinetochores during mitotic entry and for sustained spindle checkpoint arrest. CDK1-CCNB1 is therefore an integral component of the spindle checkpoint, ensuring the fidelity of mitosis.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Rata_CurrOpin_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Nov 15, 2018</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/rata_2018_currbiol/"><h4>Two Interlinked Bistable Switches Govern Mitotic Control in Mammalian Cells</h4></a>
      <h5 class="subheading">
        Scott Rata, Maria F. Suarez Peredo Rodriguez, Stephy Joseph, Nisha Peter, Fabio E. Iturra, Fengwei Yang, Anotida Madzvamuse, Jan G. Ruppert, Kumiko Samejima, Melpomeni Platani, Monica Alvarez-Fernandez, Marcos Malumbres, William C. Earnshaw, Bela Novak, Helfrid Hochegger
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2018.09.059" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2018.09.059</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Distinct protein phosphorylation levels in interphase and M phase require tight regulation of Cdk1 activity [1,2]. A bistable switch, based on positive feedback in the Cdk1 activation loop, has been proposed to generate different thresholds for transitions between these cell-cycle states [3–5]. Recently, the activity of the major Cdk1-counteracting phosphatase, PP2A:B55, has also been found to be bistable due to Greatwall kinase-dependent regulation [6]. However, the interplay of the regulation of Cdk1 and PP2A:B55 in vivo remains unexplored. Here, we combine quantitative cell biology assays with mathematical modeling to explore the interplay of mitotic kinase activation a nd phosphatase inactivation in human cells. By measuring mitotic entry and exit thresholds using ATP-analog-sensitive Cdk1 mutants, we find evidence that the mitotic switch displays  hysteresis  and  bistability,  responding differentially to Cdk1 inhibition in the mitotic and interphase states. Cdk1 activation by Wee1/Cdc25 feedback loops and PP2A:B55 inactivation by Greatwall independently contributes to this hysteretic switch system. However, elimination of both Cdk1 and PP2A:B55 inactivation fully abrogates bistability, suggesting that hysteresis is an emergent property of mutual inhibition between the Cdk1 and PP2A:B55 feedback loops. Our model of the two interlinked feedback systems predicts an intermediate but hidden steady state between interphase and M phase. This could be verified experimentally by Cdk1 inhibition during mitotic entry, supporting the predictive value of our model. Furthermore, we demonstrate that dual inhibition of Wee1 and Gwl kinases causes loss of cell-cycle memory and synthetic lethality, which could be further exploited therapeutically.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_PlosComputBiol_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Oct 24, 2018</h6>
      <h5 class="subheading">PLOS Computational Biology</h5>
      <a href="/publication/heldt_plos_2018/"><h4>Dilution and titration of cell-cycle regulators may control cell size in budding yeast</h4></a>
      <h5 class="subheading">
        Stefan Heldt, Reece Lunstone, John J. Tyson, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1371/journal.pcbi.1006548" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1371/journal.pcbi.1006548</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2018b_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2018b_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The size of a cell sets the scale for all biochemical processes within it, thereby affecting cellular fitness and survival. Hence, cell size needs to be kept within certain limits and relatively constant over multiple generations. However, how cells measure their size and use this information to regulate growth and division remains controversial. Here, we present two mechanistic mathematical models of the budding yeast (S. cerevisiae) cell cycle to investigate competing hypotheses on size control: inhibitor dilution and titration of nuclear sites. Our results suggest that an inhibitor-dilution mechanism, in which cell growth dilutes the transcriptional inhibitor Whi5 against the constant activator Cln3, can facilitate size homeostasis. This is achieved by utilising a positive feedback loop to establish a fixed size threshold for the START transition, which efficiently couples cell growth to cell cycle progression. Yet, we show that inhibitor dilution cannot reproduce the size of mutants that alter the cell’s overall ploidy and WHI5 gene copy number. By contrast, size control through titration of Cln3 against a constant number of genomic binding sites for the transcription factor SBF recapitulates both size homeostasis and the size of these mutant strains. Moreover, this model produces an imperfect ‘sizer’ behaviour in G1 and a ‘timer’ in S/G2/M, which combine to yield an ‘adder’ over the whole cell cycle; an observation recently made in experiments. Hence, our model connects these phenomenological data with the molecular details of the cell cycle, providing a systems-level perspective of budding yeast size control.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/hochegger_molecell_pub_image.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Aug 1, 2018</h6>
      <h5 class="subheading">Molecular Cell</h5>
      <a href="/publication/hochegger_molcell_2018/"><h4>DNA Replication Determines Timing of Mitosis by Restricting CDK1 and PLK1 Activation</h4></a>
      <h5 class="subheading">
        Bennie Lemmens, Nadia Hegarat, Karen Akopyan, Joan Sala-Gaston, Jiri Bartek, Helfrid Hochegger, Arne Lindqvist
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.molcel.2018.05.026" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.molcel.2018.05.026</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>To maintain genome stability, cells need to replicate their DNA before dividing. Upon completion of bulk DNA synthesis, the mitotic kinases CDK1 and PLK1 become active and drive entry into mitosis. Here, we have tested the hypothesis that DNA replication determines the timing of mitotic kinase activation. Using an optimized double-degron system, together with kinase inhibitors to enforce tight inhibition of key proteins, we find that human cells unable to initiate DNA replication prematurely enter mitosis. Preventing DNA replication licensing and/or firing causes prompt activation of CDK1 and PLK1 in S phase. In the presence of DNA replication, inhibition of CHK1 and p38 leads to premature activation of mitotic kinases, which induces severe replication stress. Our results demonstrate that, rather than merely a cell cycle output, DNA replication is an integral signaling component that restricts activation of mitotic kinases. DNA replication thus functions as a brake that determines cell cycle duration.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Novak_CurrOpin_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Feb 22, 2018</h6>
      <h5 class="subheading">Current Opinion in Systems Biology</h5>
      <a href="/publication/novak_curropinsysbiol_2018/"><h4>Genome Stability during Cell Proliferation: A Systems Analysis of the Molecular Mechanisms Controlling Progression through the Eukaryotic Cell Cycle</h4></a>
      <h5 class="subheading">
        Bela Novak, Stefan Heldt, John J. Tyson
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.coisb.2018.02.004" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.coisb.2018.02.004</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Well-nourished cells in a favorable environment (well supplied with growth factors, cytokines, and/or hormones and free from stresses, ionizing radiation, etc.) will grow, replicate their genome, and divide into two daughter cells, fully prepared to repeat the process. This cycle of DNA replication and division underlies all aspects of biological growth, reproduction, repair and development. As such, it is essential that the cell’s genome be guarded against damage during the replication/division process, lest the error(s) be irrevocably passed down to all future generations of progeny. Hence, cell cycle progression is closely guarded against major sources of errors, in particular DNA damage and misalignment of replicated chromosomes on the mitotic spindle. In this review article we examine closely the molecular mechanisms that maintain genomic integrity during the cell division cycle, and we find an unexpected and intriguing arrangement of concatenated and nested bistable toggle switches. The topology of the network seems to play crucial roles in maintaining the stability of the genome during cell proliferation.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/Heldt_Pnas_2018.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Feb 21, 2018</h6>
      <h5 class="subheading">Proceedings of the National Academy of Sciences</h5>
      <a href="/publication/heldt_pnas_2018/"><h4>A comprehensive model for the proliferation-quiescence decision in response to endogenous DNA damage in human cells</h4></a>
      <h5 class="subheading">
        Stefan Heldt, Alexis Barr, Sam Cooper, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1073/pnas.1715345115" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1073/pnas.1715345115</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2018_Heldt_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2018_Heldt_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Human cells that suffer mild DNA damage can enter a reversible state of growth arrest known as quiescence. This
decision to temporarily exit the cell cycle is essential to prevent the propagation of mutations and most cancer cells harbour
defects in the underlying control system. Here we present a mechanistic mathematical model to study the proliferation-quiescence
decision in non-transformed human cells. We show that two bistable switches, the restriction point (RP) and the G1/S transition,
mediate this decision by integrating DNA damage and mitogen signals. In particular, our data suggests that the cyclin-dependent
kinase inhibitor p21 (Cip1/Waf1), which is expressed in response to DNA damage, promotes quiescence by blocking positive feedback
loops that facilitate G1 progression downstream of serum stimulation. Intriguingly, cells exploit bistability in the RP to convert
graded p21 and mitogen signals into an all-or-nothing cell cycle response. The same mechanism creates a window of opportunity
where G1 cells that have passed the RP can revert to quiescence if exposed to DNA damage. We present experimental evidence that
cells gradually lose this ability to revert to quiescence as they progress through G1 and that the onset of rapid p21 degradation
at the G1/S transition prevents this response altogether, insulating S-phase from mild, endogenous DNA damage. Thus, two bistable
switches conspire in the early cell cycle to provide both sensitivity and robustness to external stimuli.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/hopkins_pub_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Nov 9, 2017</h6>
      <h5 class="subheading">Molecular Biology of the Cell</h5>
      <a href="/publication/hopkins_mbcell_2017/"><h4>Cell cycle transitions: a common role for stoichiometric inhibitors</h4></a>
      <h5 class="subheading">
        Michael Hopkins, John Tyson, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1091/mbc.E17-06-0349" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1091/mbc.E17-06-0349</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The cell division cycle is the process by which eukaryotic cells replicate their chromosomes and partition them to two daughter cells. To maintain the integrity of the genome, proliferating cells must be able to block progression through the division cycle at key transition points (called “checkpoints”) if there have been problems in the replication of the chromosomes or their biorientation on the mitotic spindle. These checkpoints are governed by protein-interaction networks, composed of phase-specific cell-cycle activators and inhibitors. Examples include Cdk1:Clb5 and its inhibitor Sic1 at the G1/S checkpoint in budding yeast, APC:Cdc20 and its inhibitor MCC at the mitotic checkpoint, and PP2A:B55 and its inhibitor, alpha-endosulfine, at the mitotic-exit checkpoint. Each of these inhibitors is a substrate as well as a stoichiometric inhibitor of the cell-cycle activator. Because the production of each inhibitor is promoted by a regulatory protein that is itself inhibited by the cell-cycle activator, their interaction network presents a regulatory motif characteristic of a “feedback-amplified domineering substrate” (FADS). We describe how the FADS motif responds to signals in the manner of a bistable toggle switch, and then we discuss how this toggle switch accounts for the abrupt and irreversible nature of three specific cell-cycle checkpoints.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/novak_cellcycle_2017.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Tuesday, Sep 19, 2017</h6>
      <h5 class="subheading">Cell Cycle</h5>
      <a href="/publication/novak_cellcycle_2017/"><h4>Interlinked bistable mechanisms generate robust mitotic transitions</h4></a>
      <h5 class="subheading">
        Lukas Hutter, Scott Rata, Helfrid Hochegger, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1080/15384101.2017.1371885" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1080/15384101.2017.1371885</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The transitions between phases of the cell cycle have evolved to be robust and switch-like, which ensures temporal separation of DNA replication, sister chromatid separation, and cell division. Mathematical models describing the biochemical interaction networks of cell cycle regulators attribute these properties to underlying bistable switches, which inherently generate robust, switch-like, and irreversible transitions between states. We have recently presented new mathematical models for two control systems that regulate crucial transitions in the cell cycle: mitotic entry and exit and the mitotic checkpoint.Each of the two control systems is characterized by two interlinked bistable switches. In the case of mitotic checkpoint control, these switches are mutually activating, whereas in the case of the mitotic entry/exit network, the switches are mutually inhibiting. In this Perspective we describe the qualitative features of these regulatory motifs and show that having two interlinked bistable mechanisms further enhances robustness and irreversibility. We speculate that these network motifs also underlie other cell cycle transitions and cellular transitions between distinct biochemical states.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/cooper_bioinformatics_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Tuesday, Jun 20, 2017</h6>
      <h5 class="subheading">Bioinformatics</h5>
      <a href="/publication/cooper_bioinformatics_2017/"><h4>NucliTrack: An integrated nuclei tracking application</h4></a>
      <h5 class="subheading">
        Sam Cooper, Alexis Barr, Robert Glen, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1093/bioinformatics/btx404" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1093/bioinformatics/btx404</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/samocooper/nuclitrack" target="_blank" title="View Code Resource on Github">https://github.com/samocooper/nuclitrack</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Live imaging studies give unparalleled insight into dynamic single cell behaviours and fate decisions. However, the challenge of reliably tracking single cells over long periods of time limits both the throughput and ease with which such studies can be performed. Here, we present NucliTrack, a cross platform solution for automatically segmenting, tracking and extracting features from fluorescently-labelled nuclei. NucliTrack performs similarly to other state-of-the-art cell tracking algorithms, but NucliTrack’s interactive, graphical interface makes it significantly more user friendly.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/rattani_paper_image.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, May 22, 2017</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/rattani_currbiol_2017/"><h4>APC/C:Cdh1 Enables Removal of Shugoshin-2 from the Arms of Bivalent Chromosomes by Moderating Cyclin-Dependent Kinase Activity</h4></a>
      <h5 class="subheading">
        Ahmed Rattani, Randy Ballesteros Mejia, Katherine Roberts, Maurici B. Roig, Jonathan Godwin, Michael Hopkins, Manuel Eguren, Luis Sanchez-Pulido, Elwy Okaz, Sugako Ogushi, Magda Wolna, Jean Metson, Alberto M. Pendas, Marcos Malumbres, Bela Novak, Mary Herbert, Kim Nasmyth
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2017.04.023" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2017.04.023</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>In mammalian females, germ cells remain arrested as primordial follicles. Resumption of meiosis is heralded by germinal vesicle breakdown, condensation of chromosomes, and their eventual alignment on metaphase plates. At the first meiotic division, anaphase-promoting complex/cyclosome associated with Cdc20 (APC/C<sup>Cdc20</sup>) activates separase and thereby destroys cohesion along chromosome arms. Because cohesion around centromeres is protected by shugoshin-2, sister chromatids remain attached through centromeric/pericentromeric cohesin. We show here that, by promoting proteolysis of cyclins and Cdc25B at the germinal vesicle (GV) stage, APC/C associated with the Cdh1 protein (APC/C<sup>Cdh1</sup>) delays the increase in Cdk1 activity, leading to germinal vesicle breakdown (GVBD). More surprisingly, by moderating the rate at which Cdk1 is activated following GVBD, APC/C<sup>Cdh1</sup> creates conditions necessary for the removal of shugoshin-2 from chromosome arms by the Aurora B/C kinase, an event crucial for the efficient resolution of chiasmata.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_naturecommunications_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Mar 20, 2017</h6>
      <h5 class="subheading">Nature Communications</h5>
      <a href="/publication/barr_naturecommunications_2017/"><h4>DNA damage during S-phase mediates the proliferation-quiescence decision in the subsequent G1 via p21 expression</h4></a>
      <h5 class="subheading">
        Alexis Barr, Samuel Cooper, Stefan Heldt, Francesca Butera, Henriette Stoy, Jörg Mansfeld, Bela Novak, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1038/ncomms14728" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1038/ncomms14728</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2017_Barr_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2017_Barr_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Following DNA damage caused by exogenous sources, such as ionising radiation, the tumour suppressor p53 mediates cell cycle arrest via expression of the CDK inhibitor, p21. However, the role of p21 in maintaining genomic stability in the absence of exogenous DNA damaging agents is unclear. Here, using live single cell measurements of p21 protein in proliferating cultures, we show that naturally-occurring DNA damage incurred over S-phase causes p53-dependent accumulation of p21 during mother G2- and daughter G1-phases. High p21 levels mediate G1 arrest via CDK inhibition, yet lower levels have no impact on G1 progression, and the ubiquitin ligases CRL4<sup>Cdt2</sup> and SCF<sup>Skp2</sup> couple to degrade p21 prior to the G1/S transition. Mathematical modelling reveals that a bistable switch, created by CRL4<sup>Cdt2</sup>, promotes irreversible S-phase entry by keeping p21 levels low, preventing premature S-phase exit upon DNA damage. Thus, we characterise how p21 regulates the proliferation-quiescence decision to maintain genomic stability.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/mochida_currbiol_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Nov 23, 2016</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/mochida_currbiol_2016/"><h4>Two Bistable Switches Govern M Phase Entry</h4></a>
      <h5 class="subheading">
        Satoru Mochida, Scott Rata, Hirotsugu Hino, Takeharu Nagai, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2016.10.022" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2016.10.022</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The abrupt and irreversible transition from interphase to M phase is essential to separate DNA replication from chromosome segregation. This transition requires the switch-like phosphorylation of hundreds of proteins by the cyclin-dependent kinase 1 (Cdk1):cyclin B (CycB) complex. Previous studies have ascribed these switch-like phosphorylations to the auto-activation of Cdk1:CycB through the removal of inhibitory phosphorylations on Cdk1-Tyr15 [1 and 2]. The positive feedback in Cdk1 activation creates a bistable switch that makes mitotic commitment irreversible [2, 3 and 4]. Here, we surprisingly find that Cdk1 auto-activation is dispensable for irreversible, switch-like mitotic entry due to a second mechanism, whereby Cdk1:CycB inhibits its counteracting phosphatase (PP2A:B55). We show that the PP2A:B55-inhibiting Greatwall (Gwl)-endosulfine (ENSA) pathway is both necessary and sufficient for switch-like phosphorylations of mitotic substrates. Using purified components of the Gwl-ENSA pathway in a reconstituted system, we found a sharp Cdk1 threshold for phosphorylation of a luminescent mitotic substrate. The Cdk1 threshold to induce mitotic phosphorylation is distinctly higher than the Cdk1 threshold required to maintain these phosphorylations—evidence for bistability. A combination of mathematical modeling and biochemical reconstitution show that the bistable behavior of the Gwl-ENSA pathway emerges from its mutual antagonism with PP2A:B55. Our results demonstrate that two interlinked bistable mechanisms provide a robust solution for irreversible and switch-like mitotic entry.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/cundell_jcb_2016.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Aug 22, 2016</h6>
      <h5 class="subheading">Journal of Cell Biology</h5>
      <a href="/publication/cundell_jcb_2016/"><h4>A PP2A-B55 recognition signal controls substrate dephosphorylation kinetics during mitotic exit</h4></a>
      <h5 class="subheading">
        Michael Cundell, Lukas Hutter, Ricardo Nunes-Bastos, Elena Poser, James Holder, Shabaz Mohammed, Bela Novak, Francis Barr
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1083/jcb.201606033" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1083/jcb.201606033</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2016_Cundell_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2016_Cundell_et_al</a>
              </div>
            </div>
          </li>
        
        
          <li>
            <div class="collapsible-header">
              <i class="fa fa-area-chart"></i>
              <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html" target="_blank" title="Visualisation Resource">View Datavisualisation</a>
            </div>
          </li>
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>PP2A-B55 is one of the major phosphatases regulating cell division. Despite its importance for temporal control during mitotic exit, how B55 substrates are recognized and differentially dephosphorylated is unclear. Using phosphoproteomics combined with kinetic modeling to extract B55-dependent rate constants, we have systematically identified B55 substrates and assigned their temporal order in mitotic exit. These substrates share a bipartite polybasic recognition determinant (BPR) flanking a Cdk1 phosphorylation site. Experiments and modeling show that dephosphorylation rate is encoded into B55 substrates, including its inhibitor ENSA, by cooperative action of basic residues within the BPR. A complementary acidic surface on B55 decodes this signal, supporting a cooperative electrostatic mechanism for substrate selection. A further level of specificity is encoded into B55 substrates because B55 displays selectivity for phosphothreonine. These simple biochemical properties, combined with feedback control of B55 activity by the phosphoserine-containing substrate/inhibitor ENSA, can help explain the temporal sequence of events during exit from mitosis.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/hegarat_bioessays_2016.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Friday, May 27, 2016</h6>
      <h5 class="subheading">BioEssays</h5>
      <a href="/publication/hegarat_bioessays_2016/"><h4>Bistability of mitotic entry and exit switches during open mitosis in mammalian cells</h4></a>
      <h5 class="subheading">
        Nadia Hegarat, Scott Rata, Helfrid Hochegger
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1002/bies.201600057" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1002/bies.201600057</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Mitotic entry and exit are switch-like transitions that are driven by the activation and inactivation of Cdk1 and mitotic cyclins. This simple on/off reaction turns out to be a complex interplay of various reversible reactions, feedback loops, and thresholds that involve both the direct regulators of Cdk1 and its counteracting phosphatases. In this review, we summarize the interplay of the major components of the system and discuss how they work together to generate robustness, bistability, and irreversibility. We propose that it may be beneficial to regard the entry and exit reactions as two separate reversible switches that are distinguished by differences in the state of phosphatase activity, mitotic proteolysis, and a dramatic rearrangement of cellular components after nuclear envelope breakdown, and discuss how the major Cdk1 activity thresholds could be determined for these transitions.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/chica_currbiol_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Feb 8, 2016</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/publication/chica_currbiol_2016/"><h4>Nutritional Control of Cell Size by the Greatwall-Endosulfine-PP2A·B55 Pathway</h4></a>
      <h5 class="subheading">
        Nathalia Chica, Ana Rozalen, Livia Perez-Hidalgo, Angela Rubio, Bela Novak, Sergio Moreno
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2015.12.035" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2015.12.035</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Proliferating cells adjust their cell size depending on the nutritional environment. Cells are large in rich media and small in poor media. This physiological response has been demonstrated in both unicellular and multicellular organisms. Here we show that the greatwall-endosulfine (Ppk18-Igo1 in fission yeast) pathway couples the nutritional environment to the cell-cycle machinery by regulating the activity of PP2A·B55. In the presence of nutrients, greatwall (Ppk18) protein kinase is inhibited by TORC1 and PP2A·B55 is active. High levels of PP2A·B55 prevent the activation of mitotic Cdk1·Cyclin B, and cells increase in size in G2 before they undergo mitosis. When nutrients are limiting, TORC1 activity falls off, and the activation of greatwall (Ppk18) leads to the phosphorylation of endosulfine (Igo1) and inhibition of PP2A·B55, which in turn allows full activation of Cdk1·CyclinB and entry into mitosis with a smaller cell size. Given the conservation of this pathway, it is reasonable to assume that this mechanism operates in higher eukaryotes, as well.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/barr_cellsystems_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 27, 2016</h6>
      <h5 class="subheading">Cell Systems</h5>
      <a href="/publication/barr_cellsystems_2016/"><h4>A Dynamical Framework for the All-or-None G1/S Transition</h4></a>
      <h5 class="subheading">
        Alexis Barr, Stefan Heldt, Tongli Zhang, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cels.2016.01.001" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cels.2016.01.001</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The transition from G1 into DNA replication (S phase) is an emergent behavior resulting from dynamic and complex interactions between cyclin-dependent kinases (Cdks), Cdk inhibitors (CKIs), and the anaphase-promoting complex/cyclosome (APC/C). Understanding the cellular decision to commit to S phase requires a quantitative description of these interactions. We apply quantitative imaging of single human cells to track the expression of G1/S regulators and use these data to parametrize a stochastic mathematical model of the G1/S transition. We show that a rapid, proteolytic, double-negative feedback loop between Cdk2:Cyclin and the Cdk inhibitor p27Kip1 drives a switch-like entry into S phase. Furthermore, our model predicts that increasing Emi1 levels throughout S phase are critical in maintaining irreversibility of the G1/S transition, which we validate using Emi1 knockdown and live imaging of G1/S reporters. This work provides insight into the general design principles of the signaling networks governing the temporally abrupt transitions between cell-cycle phases.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/mirkovic_cellreports_2015.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Oct 8, 2015</h6>
      <h5 class="subheading">Cell Reports</h5>
      <a href="/publication/mirkovic_cellreports_2015/"><h4>Premature Sister Chromatid Separation Is Poorly Detected by the Spindle Assembly Checkpoint as a Result of System-Level Feedback</h4></a>
      <h5 class="subheading">
        Mihailo Mirkovic, Lukas Hutter, Bela Novak, Raquel Oliveira
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.celrep.2015.09.020" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.celrep.2015.09.020</a></div>
          </div>
        </li>
        
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2015_Mirkovic_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2015_Mirkovic_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Sister chromatid cohesion, mediated by the cohesin complex, is essential for faithful mitosis. Nevertheless, evidence suggests that the surveillance mechanism that governs mitotic fidelity, the spindle assembly checkpoint (SAC), is not robust enough to halt cell division when cohesion loss occurs prematurely. The mechanism behind this poor response is not properly understood. Using developing Drosophila brains, we show that full sister chromatid separation elicits a weak checkpoint response resulting in abnormal mitotic exit after a short delay. Quantitative live-cell imaging approaches combined with mathematical modeling indicate that weak SAC activation upon cohesion loss is caused by weak signal generation. This is further attenuated by several feedback loops in the mitotic signaling network. We propose that multiple feedback loops involving cyclin-dependent kinase 1 (Cdk1) gradually impair error-correction efficiency and accelerate mitotic exit upon premature loss of cohesion. Our findings explain how cohesion defects may escape SAC surveillance.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/tyson_bmcbiology_2015.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jul 1, 2015</h6>
      <h5 class="subheading">BMC Biology</h5>
      <a href="/publication/tyson_bmcbiology_2015/"><h4>Models in biology: lessons from modeling regulation of the eukaryotic cell cycle</h4></a>
      <h5 class="subheading">
        John Tyson, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1186/s12915-015-0158-9" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1186/s12915-015-0158-9</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>In this essay we illustrate some general principles of mathematical modeling in biology by our experiences in studying the molecular regulatory network underlying eukaryotic cell division. We discuss how and why the models moved from simple, parsimonious cartoons to more complex, detailed mechanisms with many kinetic parameters. We describe how the mature models made surprising and informative predictions about the control system that were later confirmed experimentally. Along the way, we comment on the ‘parameter estimation problem’ and conclude with an appeal for a greater role for mathematical models in molecular cell biology.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
              
              <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="http://cellcycle.org.uk//img/publication/pk_febsletters_2015.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Feb 12, 2015</h6>
      <h5 class="subheading">FEBS Letters</h5>
      <a href="/publication/pk_febsletters_2015/"><h4>Model scenarios for switch-like mitotic transitions</h4></a>
      <h5 class="subheading">
        Vinod PK, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.febslet.2015.02.007" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.febslet.2015.02.007</a></div>
          </div>
        </li>
        
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>To facilitate rapid accumulation of Cdk1-phosphorylated substrate proteins, the Cdk1 counter-acting phosphatase, PP2A-B55 is inhibited during M phase by stoichiometric inhibitors (ENSA and Arpp19). These inhibitors are activated when phosphorylated by Cdk1-activated Greatwall-kinase. Recent experiments show that ENSA is dephosphorylated and inactivated by the PP2A-B55 itself, and acts as an unfair substrate inhibiting PP2A-B55 activity towards other Cdk1 substrates. Mathematical modelling shows that this mutual antagonism between the phosphatase and its inhibitor is insufficient to explain the switch-like characteristics of mitotic entry and exit. We show that the feedback regulation of Greatwall activating kinase and/or inactivating phosphatase can explain the abruptness of these cell cycle transitions.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

            
          </ul>
        </div>
      </div>
    </section>

    

    <!-- jQuery -->
<script src="http://cellcycle.org.uk/js/jquery-v3.3.1/jquery.min.js"></script>

<!-- Bootstrap Core JavaScript -->
<script src="http://cellcycle.org.uk/js/bootstrap-v3.3.7/bootstrap.js"></script>

<!-- Plugin JavaScript -->
<script src="http://cellcycle.org.uk/js/classie-v1.0.1.js"></script>
<script src="http://cellcycle.org.uk/js/cbpAnimatedHeader-v1.0.0.js"></script>

<!-- Contact Form JavaScript -->
<script src="http://cellcycle.org.uk/js/jqBootstrapValidation-v1.3.6-dirty.js"></script>
<script src="http://cellcycle.org.uk/js/contact_me.js"></script>

<!-- Custom Theme JavaScript -->
<script src="http://cellcycle.org.uk/js/agency.js"></script>

<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
<script type="text/javascript" src="http://cellcycle.org.uk/js/materialize.min.js"></script>







  </body>
</html>
